io-engine/src/logger.rs

```diff
@@ -380,6 +391,21 @@ impl LogFormat {
 
         writeln!(writer)
     }
+
+    fn hostname(&self) -> String {
+        if self.show_host {
+            let mut buf = [0u8; 64];
+            let s = match nix::unistd::gethostname(&mut buf) {
+                Ok(name) => name.to_str().unwrap_or("").to_string(),
+                Err(_) => {
+                    std::env::var("HOSTNAME").unwrap_or_else(|_| String::new())
+                }
+            };
+            format!("{} :: ", s)
```

Comment:
does it need space? Would you mind show me what the logs look like?

Reply:
Like this:
`ms_src_0 :: 18:56:13.868020 | env.rs:836         [I] Total number of cores available: 2
ms_src_0 :: 18:56:13.882829 | reactor.rs:182     [I] Scheduled SPDK thread 'init_thread' (0x55af895714f0) on core #5
ms_src_0 :: 18:56:13.882927 | reactor.rs:158     [I] Init thread ID 1
ms_src_0 :: 18:56:13.883024 | reactor.rs:301     [I] Starting reactor polling loop core=6 tid=9
ms_src_0 :: 18:56:13.884034 | env.rs:865         [I] All cores locked and loaded!
ms_src_0 :: 18:56:13.884170 | accel_engine.c:969 [D] Accel framework software engine initialized.   
ms_src_0 :: 18:56:13.884209 | mod.rs:58          [D] mayastor subsystem init
ms_src_0 :: 18:56:13.938127 | vbdev_crypto.c:461 [D] Found crypto devices: 1   
`

Reply:
Message prefix is hostname, following by ::, then date&time, pipe, src filename truncated, level.
It is fixed size, so it is visually easier to follow the log. When colorized, the entire line color indicates the log level.